### ISSUE TITLE
[FEATURE] some features from xs camera

### ISSUE THREAD

**Issue by rajarshikhatua** [anonfaded/FadCam#82]
https://play.google.com/store/apps/details?id=me.jagar.xscamera2

from this app specifically, shedule camera, change icon, live streaming, mini preview

**Comment by anonfaded** [comment]
Yes all these will be added in upcoming releases gradually. Thanks
